aea/aea_builder.py

```diff
@@ -743,19 +724,20 @@ def build(
         :param ledger_apis: the api ledger that we want to use.
         :return: the AEA object.
         """
-        wallet = Wallet(self.private_key_paths)
+        resources = Resources()
+        wallet = Wallet(deepcopy(self.private_key_paths))
         identity = self._build_identity_from_wallet(wallet)
-        ledger_apis = self._load_ledger_apis(ledger_apis)
-        self._load_and_add_components(ComponentType.PROTOCOL)
-        self._load_and_add_components(ComponentType.CONTRACT)
+        ledger_apis = deepcopy(self._load_ledger_apis(ledger_apis))
```

Comment:
Could move 'deepcopy' into method

Reply:
what do you mean? A wrapper to `deepcopy`? I don't see much benefits.

Reply:
No, I mean use deepcopy in the method being called rather than here, just for symmetry with the rest, not crucial though.

Reply:
ah! Sorry. Yes, good point.